File: backend/Models/Employee.cs
namespace backend.models;


public class Employee {

    public long id {get; set;}
    public string Name {get; set;}

    public string LastName {get; set;}

    public string? Address {get; set;}

    public DateOnly DateOfBirth {get; set;}

    public DateOnly StartingDate {get;set;}

    public decimal Salary {get; set;}

    public virtual ICollection<PositionsEmployee> Positions {get; set;}
}

File: frontend/zadanie/src/models/employee.ts
import type {Positions} from '@/models/position'
type EmployeePosition ={
    id : Number;
    positionId : Number;
    position : Positions;
    start : string;
    end : string;
}

export class Employee {
    id: Number;
    name: string;
    lastName: string;
    salary: Number;
    dateOfBirth: string;
    address: string;
    startingDate : string
    positions : EmployeePosition[]

    constructor(){
        this.id = -1
        this.name = ""
        this.lastName =""
        this.salary = 0
        this.dateOfBirth = ""
        this.address = ""
        this.startingDate = ""
        this.positions = new Array<EmployeePosition>
    }
}